pkg/plugins/resources/k8s/store_test.go

```diff
@@ -560,6 +560,37 @@ var _ = Describe("KubernetesStore", func() {
 			Expect(err).ToNot(HaveOccurred())
 			Expect(actual.Meta.GetLabels()[mesh_proto.DisplayName]).To(Equal("dn"))
 		})
+
+		It("should not add namespace label when resource is synced from zone", func() {
+			// setup
+			expected := backend.ParseYAML(fmt.Sprintf(`
+            apiVersion: kuma.io/v1alpha1
+            kind: TrafficRoute
```

Comment:
please use a new policy for the test, so we have one less test to rewrite when we remove old policies

Reply:
changed